tests/integration/test_join_set_family_s3/configs/minio.xml

```diff
@@ -6,7 +6,7 @@
                 <endpoint>http://minio1:9001/root/data/</endpoint>
                 <access_key_id>minio</access_key_id>
                 <secret_access_key>minio123</secret_access_key>
-                <send_object_metadata>true</send_object_metadata>
+                <send_metadata>true</send_metadata>
```

Comment:
may just remove this config setting from here? it is anyway not covered by the tests